llvm/test/CodeGen/RISCV/xmips-exectl.ll

```diff
@@ -0,0 +1,35 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --tool /home/ukalappa/llvm/github/build/bin/llc --version 6
```

Comment:
if you use `--llc-binary` instead it won't attach `--tool <path>` here I believe.